internal/app/machined/pkg/runtime/v1alpha1/v1alpha1_sequencer_tasks.go

```diff
@@ -575,6 +576,32 @@ func StartContainerd(seq runtime.Sequence, data interface{}) runtime.TaskExecuti
 	}
 }
 
+// StartUdevd represents the task to start udevd.
+func StartUdevd(seq runtime.Sequence, data interface{}) runtime.TaskExecutionFunc {
+	return func(ctx context.Context, logger *log.Logger, r runtime.Runtime) (err error) {
+		svc := &services.Udevd{}
+
+		system.Services(r).LoadAndStart(svc)
+
+		ctx, cancel := context.WithTimeout(ctx, 5*time.Minute)
+		defer cancel()
+
+		if err = system.WaitForService(system.StateEventUp, svc.ID(r)).Wait(ctx); err != nil {
+			return err
+		}
+
+		if _, err = cmd.Run("/sbin/udevadm", "trigger"); err != nil {
+			return err
+		}
+
+		if _, err = cmd.Run("/sbin/udevadm", "settle"); err != nil {
```

Comment:
This ensures that `udevd` finishes processing kernel events triggered by `udevd trigger`. This prevents a race condition between `udevd` and Talos when a user wants to use something under `/dev/disk/*` in `machine.disks`.